.github/workflows/pythonpackage.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
If dropping 3.6 from CI, should we specify the Python versions in the setup.py with `python_requires='>=3.7.*'`?